bitnami/mariadb/templates/serviceaccount.yaml

```diff
@@ -15,4 +15,5 @@ metadata:
     {{- if .Values.commonAnnotations }}
     {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
     {{- end }}
+automountServiceAccountToken: {{ .Values.serviceAccount.automountServiceAccountToken }}
```

Comment:
AFAIK, none of the containers in this chart access the API server. Then, I don't see why it's necessary to expose this parameter, it's useless.

Reply:
Forget about this comment @migruiz4 ! I was talking about this point with @javsalgar and it could be useful if users add custom initcontainer/sidecars that talk with the K8s API.
That said, you're currently setting it to `true` by default, please do the opposite and set it to `false`. I mean, there's no reason to mount the token in the container UNLESS you need access the to the API